valkey.conf

```diff
@@ -1771,6 +1771,14 @@ aof-timestamp-enabled no
 #
 # cluster-preferred-endpoint-type ip
 
+# The cluster blacklist is useful when removing a node from the cluster completely. 
+# When CLUSTER FORGET is called for a node, that node is put into the blacklist for
+# some time so that when gossip messages are received from other nodes that still
+# remember it, it is not re-added. This gives time for CLUSTER FORGET to be sent to
+# every node in the cluster. The blacklist TTL is 60 seconds by default.
```

Comment:
```suggestion
# The cluster blacklist is used when removing a node from the cluster completely. 
# When CLUSTER FORGET is called for a node, that node is put into the blacklist for
# some time so that when gossip messages are received from other nodes that still
# remember it, it is not re-added. This gives time for CLUSTER FORGET to be sent to
# every node in the cluster. The blacklist TTL is 60 seconds by default, which should
# sufficient for most clusters, but you may considering increasing this if you see
# nodes getting re-added while using CLUSTER FORGET.
```

Just make it a bit more prescriptive.